docs/api/user.rst

```diff
@@ -92,6 +92,16 @@ Here we maintain a few package specific utilities for feature selection, etc.
    data.organize_cite_seq_10x
    data.organize_multiome_anndatas
 
+Utilities
+~~~~~~~~~~
+
+Here we maintain a miscellaneous general methods.
```

Comment:
```suggestion
Here we maintain miscellaneous general methods.
```

or have i been using miscellaneous wrong my whole life

Reply:
no lol

Reply:
it's me